### ISSUE TITLE
Cannot retrieve a document with revs_info

### ISSUE THREAD

**Issue by dtelyukh** [go-kivik/kivik#509]
1. Which version of Kivik are you using? Please state a version number, or `master`.
`3.2.0`

2. Which version of Go are you using?
`go version go1.15.2 linux/amd64`

3. What did you do?
```
	couchDBClient, err := kivik.New("couch", myHost)
	if err != nil {
		log.Fatal(err)
	}
	defer couchDBClient.Close(context.Background())

	couchDB := couchDBClient.DB(context.Background(), myDb)
	defer couchDB.Close(context.Background())

	options := make(map[string]interface{})
	options["revs_info"] = true
	row := couchDB.Get(context.Background(), docId, options)

	doc := map[string]interface{}{}
	err = row.ScanDoc(&doc)

	if err != nil {
		log.Fatal(err)
	}

	for fieldName, fieldValue := range doc {
		log.Printf("%s:%v", fieldName, fieldValue)
	}
```

4. What did you expect to see?
Retrieved document

5. What did you see instead?
`no ETag header found`

**Comment by flimzy** [comment]
Thanks for the bug report. I have been able to reproduce this.  I'll work on a fix shortly.

**Comment by flimzy** [comment]
This should be fixed in `go-kivik/couchdb` [v3.2.1](https://github.com/go-kivik/couchdb/releases/tag/v3.2.1) now:

```bash
go get github.com/go-kivik/couchb/v3@v3.2.1
```

Please let me know if this doesn't solve it for you.
